contracts/access/SignatureBouncer.sol

```diff
@@ -9,22 +9,22 @@ import "../ECRecovery.sol";
  * @title SignatureBouncer
  * @author PhABC, Shrugs and aflesher
  * @dev Bouncer allows users to submit a signature as a permission to do an action.
- * @dev If the signature is from one of the authorized bouncer addresses, the signature
- * @dev is valid. The owner of the contract adds/removes bouncers.
- * @dev Bouncer addresses can be individual servers signing grants or different
- * @dev users within a decentralized club that have permission to invite other members.
- * @dev
- * @dev This technique is useful for whitelists and airdrops; instead of putting all
- * @dev valid addresses on-chain, simply sign a grant of the form
- * @dev keccak256(`:contractAddress` + `:granteeAddress`) using a valid bouncer address.
- * @dev Then restrict access to your crowdsale/whitelist/airdrop using the
- * @dev `onlyValidSignature` modifier (or implement your own using isValidSignature).
- * @dev
- * @dev In addition to `onlyValidSignature`, `onlyValidSignatureAndMethod` and
- * @dev `onlyValidSignatureAndData` can be used to restrict access to only a given method
- * @dev or a given method with given parameters respectively.
- * @dev
- * @dev See the tests Bouncer.test.js for specific usage examples.
+ * If the signature is from one of the authorized bouncer addresses, the signature
+ * is valid. The owner of the contract adds/removes bouncers.
+ * Bouncer addresses can be individual servers signing grants or different
+ * users within a decentralized club that have permission to invite other members.
+ * 
```

Comment:
I'm still not sure how natspec is defined for multiline strings and also those with newlines. cc @frangio do you have ideas? I think the parser should parse "from the tag until the end of the block or the beginning of the next tag" but I'm not sure how it's actually coded.

Reply:
I actually don't know, we would have to see the compiler output. Maybe @elopio has something already set up to easily test it. :)